coalib/bearlib/languages/definitions/Tcl.py

```diff
@@ -0,0 +1,12 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class Tcl:
+    aliases = 'Tcl/Tk',
+    versions = 8.0, 8.1, 8.2, 8.3, 8.4, 8.5, 8.6
+    extensions = '.tcl',
+    comment_delimiter = '#'
```

Comment:
not quite..

'#`' only works at the beginning of a command, which is conceptually incompatible with the other usages of `comment_delimiter` which assume a comment delimiter has a high precedence.

Typically this means in Tcl, `; #` is needed to be a comment occurring in the middle of a line.

This difference is quite small, and quite achievable to support, but is getting further from your project scope of quotes, so I think discard this from the MR, and please create an issue about supporting Tcl comments.

Reply:
I have created an issue regarding this here https://github.com/coala/coala/issues/5580. 

For now, I have removed `comment_delimiter` from this PR.